Core/Src/mpu.c

```diff
@@ -51,8 +65,8 @@ mpu_t *init_mpu(ADC_HandleTypeDef *pedals_adc, ADC_HandleTypeDef *lv_adc,
 	/* Initialize the Onboard Temperature Sensor */
 	mpu->temp_sensor = malloc(sizeof(sht30_t));
 	assert(mpu->temp_sensor);
-	mpu->temp_sensor->i2c_handle = hi2c;
-	assert(!sht30_init(mpu->temp_sensor)); /* This is always connected */
+	assert(!sht30_init(mpu->temp_sensor, temp_write_reg,
+			     temp_read_reg)); /* This is always connected */
 
 	assert(!HAL_ADC_Start_DMA(mpu->pedals_adc, mpu->pedal_dma_buf,
 				  sizeof(mpu->pedal_dma_buf) /
```

Comment:
this looks good to me. @jr1221 we dont have one of these on mpu right now do we? if not i say we can just merge this one ths embedded base PR looks good. If there is one of these on mpu, lets test before merge

Reply:
what kind of testing should I do? I didn't see any test files in caios work, is there a standard for testing in C that NER uses?